web/sites/default/config/webform.webform.report_cable_complaint.yml

```diff
@@ -0,0 +1,480 @@
+      'Billing/Cost (unexpected rate increase)': 'Billing/Cost (unexpected rate increase)'
+      'Billing/Cost (unexpected fee)': 'Billing/Cost (unexpected fee)'
+      'Billing/Cost (franchise fee)': 'Billing/Cost (franchise fee)'
+      'Billing/Cost (other)': 'Billing/Cost (other)'
+      'Installation/Repair (late or no show)': 'Installation/Repair (late or no show)'
+      'Installation/Repair (damaged property)': 'Installation/Repair (damaged property)'
+      'Installation/Repair (other)': 'Installation/Repair (other)'
+      'Phones (long wait time, busy signal, hang up)': 'Phones (long wait time, busy signal, hang up)'
+      'Phones (difficult to understand menu options)': 'Phones (difficult to understand menu options)'
+      'Phones (can’t speak to a human)': 'Phones (can’t speak to a human)'
+      'Phones (other)': 'Phones (other)'
+      'Service Quality (service outages/poor quality)': 'Service Quality (service outages/poor quality)'
+      'Service Quality (equipment isn’t working)': 'Service Quality (equipment isn’t working)'
+      'Service Quality (other)': 'Service Quality (other)'
+      Other: Other
+    '#required': true
+  report_issue_description:
+    '#type': textarea
+    '#title': 'Description of Issue or Concern'
+  contact_name:
+    '#type': textfield
+    '#title': 'Your Name'
+  contact_email:
+    '#type': textfield
+    '#title': 'Your Email'
+    '#required': true
+  contact_phone:
+    '#type': textfield
+    '#title': 'Your Phone Number'
+    '#required': true
+  contact_zip_code:
+    '#type': textfield
+    '#title': 'Your Zip Code'
+    '#description': 'The MHCRC tracks issues and concerns by zip code for reporting purposes.'
+    '#maxlength': 5
+    '#required': true
+    '#pattern': '^[0-9]{5}$'
+  support_agent_use_only:
+    '#type': portland_support_agent_widget
+    '#title': 'Support Agent Use Only'
+  report_ticket_id:
+    '#type': hidden
+    '#title': 'Report Ticket Id'
+  resolution_url:
+    '#type': webform_computed_twig
+    '#title': 'Resolution URL'
+    '#display_on': none
+    '#mode': text
+    '#template': '{{ webform_token(''[site:url-brief]'', webform_submission, [], options) }}/form/cable-complaint-resolution?report_service_provider={{ data.report_service_provider }}&report_account_number={{ data.report_account_number }}&report_issue_type={{ data.report_issue_type }}&contact_zip_code={{ data.contact_zip_code }}&report_issue_description={{ data.report_issue_description }}&contact_name={{ data.contact_name }}&contact_email={{ data.contact_email }}&contact_phone={{ data.contact_phone }}'
+    '#whitespace': trim
+    '#store': true
+    '#ajax': true
+  computed_recipient:
+    '#type': webform_computed_twig
+    '#title': 'Computed Recipient'
+    '#display_on': none
+    '#mode': text
+    '#template': |-
+      {% if data.report_service_provider == "Xfinity (Comcast)" %}
+      gregoryscottclapp+xfinity1@gmail.com,gregoryscottclapp+xfinity2@gmail.com
```

Comment:
Do these emails need to be swapped out for the actual emails?

Reply:
Oops, thanks for catching that! I've updated the field with the contact addresses in the docs.